src/components/App.stories.mdx

```diff
@@ -133,7 +123,7 @@ If translations are enabled on the form, a `LanguageSelector` is rendered:
     }}
     args={{
       'form.translationEnabled': true,
-      baseUrl: process.env.REACT_APP_BASE_API_URL || BASE_URL,
+      baseUrl: BASE_URL,
```

Comment:
If I change the base URL in the controls, it breaks the component. Maybe it shouldn't be a control?
![Screenshot from 2023-02-24 15-01-37](https://user-images.githubusercontent.com/19154114/221197359-53f59992-f7bd-47f3-96f1-a54339df6d9b.png)

Reply:
yeah this one should be removed, since the mocks don't update dynamically anyway :thinking: 

possibly we find a way in the future, but for now setting the envvar is likely the easiest.